website/src/theme/BlogLayout/index.js

```diff
@@ -132,7 +132,7 @@
        
        <div className="mobile-toc-section container">
          {toc && (
-           <div className="mobile-toc-container">
+           <div className="mobile-toc-,4container">
```

Comment:
Huh, thought I left a comment here already...

Is this class name (`mobile-toc-,4container`) intentional, or a typo?

Reply:
Woop, I think that's a typo! Great catch. I pushed a fix.